{"uuid": "49939af9-77cb-4dda-b15b-1a2ce65d9c3d", "children": ["e7b83066-283b-4dfb-af2b-c5d10210c4af"], "befores": [{"name": "read_config_file", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '../Config/config.json'\n", "trace": "  File \"C:\\Python39\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 1124, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 930, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\PythonSelenium\\Tests\\conftest.py\", line 17, in read_config_file\n    with open(CONFIG_PATH) as config_file:\n"}, "start": 1605455227605, "stop": 1605455227605}], "start": 1605455227605, "stop": 1605455227957}